ViewModel/Utilities/ImportFromFDA1Helper.cs

```diff
@@ -57,108 +57,110 @@ private static RatingCurveElement CreateRatingElement(RatingFunction rat)
             {
-                //not sure if this is possible. Look at the importer project class RatingFunction. I could do it but i would have to make the arrays public.
-                //if this is possible, maybe i could just set the global variables. I think the logic is the same.
-                return CreateRatingPairedDataWithPointUncertainty(rat);
+                ys = CreateNormalDistributions(rat);
+            }
+            else if (rat.ErrorTypesId == ErrorType.TRIANGULAR)
+            {
+                ys = CreateTriangularDistributions(rat);
+            }
+            else if (rat.ErrorTypesId == ErrorType.UNIFORM)
+            {
+                ys = CreateUniformDistributions(rat);
+            }
+            else if (rat.ErrorTypesId == ErrorType.NONE)
+            {
+                ys = CreateDeterministicDistributions(rat);
             }
-        }
-
-        private static UncertainPairedData CreateRatingPairedDataWithPointUncertainty(RatingFunction rat)
-        {
-            UncertainPairedData upd = new UncertainPairedData();
-            //if (rat.ErrorTypesId == ErrorType.LOGNORMAL)
-            //{
-            //    rat._st
-            //    List<Deterministic> yVals = new List<Deterministic>();
-            //    foreach (double d in ys)
-            //    {
-            //        yVals.Add(new Deterministic(d));
-            //    }
-            //    UncertainPairedData curve = new UncertainPairedData(xs.ToArray(), yVals.ToArray(), xLabel, yLabel, name, "", -1);
-            //    return curve;
-            //}
-            //else if (rat.ErrorTypesId == ErrorType.NORMAL)
-            //{
-
-            //}
-            //else if (rat.ErrorTypesId == ErrorType.TRIANGULAR)
-            //{
-
-            //}
-            //else if (rat.ErrorTypesId == ErrorType.UNIFORM)
-            //{
-
-            //}
-            //else if (rat.ErrorTypesId == ErrorType.NONE)
-            //{
 
-            //}
-            return upd;
+            return new UncertainPairedData(rat.GetStage(), ys.ToArray(), "Stage", "Flow", "Rating", "", -1);
         }
 
-        private static UncertainPairedData CreateRatingPairedDataWithGlobalUncertainty(RatingFunction rat)
+        private static List<IDistribution> CreateLogNormalDistributions(RatingFunction rat)
         {
-            UncertainPairedData upd = new UncertainPairedData();
-            string xLabel = "Stage";
-            string yLabel = "Flow";
-            string name = "Rating";
-
-            double[] stages = rat.GetStage();
-            double[] flows = rat.GetDischarge();
-
-            if (rat.ErrorTypesId == ErrorType.LOGNORMAL)
+            List<IDistribution> ys = new List<IDistribution>();
+            for (int i = 0; i < rat.NumberOfPoints; i++)
             {
-                List<IDistribution> ys = new List<IDistribution>();
-                for (int i = 0; i < rat.NumberOfPoints; i++)
+                if (rat.UsesGlobalError)
```

Comment:
Not a huge fan of the name, I'd say UsesGlobalStd or something more descriptive, but I don't know its full usage outside of these cases.

Reply:
Doing this logic on the bool property feels very arbitrary from an end user of the API standpoint.  Is there a way we can do this logic in the API at all?  Probably not, I see that we have two different types for the output.  Just a thought that's probably not actionable.

Reply:
This is coming from Bob's importer code. I don't really want to dive into that code and change. Things. Lets just leave this alone for now.